backend/schemas/dashboardTile.js

```diff
@@ -20,6 +20,8 @@ var tiles = mongoose.Schema({
 var items = mongoose.Schema({
     "itemIndex": Number,
     "displayTitle": {type: String, default: ''},
+    "tilesPerRow": {type: Number, default: 3},
```

Comment:
As it's not translatable would expect this to be `_titlesPerRow` and also `_ shouldTilesStretchAcrossRows `

Reply:
Ahhh, thank you! Will change now.